docs/index.bs

```diff
@@ -2134,23 +2104,29 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
 
       <dfn method for="Cache"><code>keys(|request|, |options|)</code></dfn> method *must* run these steps:
 
-        1. Let |promise| be a new <a>promise</a>.
-        1. Run these substeps <a>in parallel</a>:
-            1. Let |resultArray| be an empty array.
+        1. Let |r| be null.
+        1. If the optional argument |request| is not omitted, then:
+            1. If |request| is a {{Request}} object, then:
+                1. Set |r| to |request|'s [=Request/request=].
+                1. If |r|'s [=request/method=] is not \`<code>GET</code>\` and |options|.ignoreMethod is false, return [=a promise resolved with=] an empty array.
+            1. Else if |request| is a string, then:
+                1. Set |r| to the associated [=Request/request=] of the result of invoking the initial value of {{Request}} as constructor with |request| as its argument. If this [=throws=] an exception, return [=a promise rejected with=] that exception.
+        1. Let |realm| be the [=current Realm Record=].
```

Comment:
As above, we might not need this if we use frozenarray.

Reply:
In the above case, I changed it to using a frozen array but still left "in realm" when creating the frozen array. If we use frozen array here, don't we need to specify a realm?